### PR TITLE
Aws tfstate backend

### PR DESCRIPTION
**Pull request by wirewc**
**Changes:**
* Add ability to use a 3rd party S3 bucket
* Parameters to use that S3 bucket through the zarf package.

### CODE REVIEW

File: aws/zarf.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@ variables:
   description: "Provide a means to force destroy the Loki storage (S3) bucket even if it contains data"
   default: "false"
 
+- name: TFSTATE_BUCKET_NAME
+  description: "Terraform Backend S3 bucket for managing Terraform State files"
+
+- name: TFSTATE_DYNAMODB_TABLE_NAME
+  description: "Terraoform DynamoDB Lockfile database"
```
Comment (MxNxPx) [inline aws/zarf.yaml:28]: Typo in description: `Terraoform`

**Review by MxNxPx — COMMENTED** [review]
@wirewc - would you mind updating the aws [zarf-config.yaml](https://github.com/defenseunicorns/zarf-package-big-bang/blob/aws-tfstate-backend/aws/zarf-config.yaml) to reflect some of the new zarf vars please?

File: aws/zarf.yaml (hunk 1 above)
Comment (wirewc) [reply aws/zarf.yaml:28]: On it!

**Comment by wirewc** [comment]
@UncleGedd looks like he wants to help out with the `zarf.yaml` and some of the CI setup things.

**Comment by UncleGedd** [comment]
This PR has been superseded by this [commit](https://github.com/defenseunicorns/uds-package-dubbd/commit/22522f7b4954d574d4e36ac3bae67d2323a8bebd) and the decision to ingest an existing state bucket and dynamodb table for now

**Pull request closed without merging**